src/login.js

```diff
@@ -0,0 +1,122 @@
+"use strict";
```

Comment:
Your out directory should be `./lib` when you run tsc

Reply:
There we go!